### PR TITLE
Remove NPM from the list of repos where protos are published

### PR DESCRIPTION
**Pull request by tigrannajaryan**
Resolves https://github.com/open-telemetry/opentelemetry-proto/issues/379

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 The proto files can be consumed as GIT submodules or copied and built directly in the consumer project.
 
-The compiled files are published to central repositories (Maven, NPM...) from OpenTelemetry client libraries.
+The compiled files are published to central repositories (Maven, ...) from OpenTelemetry client libraries.
```
Comment (bogdandrutu) [inline README.md:7]: Not sure if it is publish to maven, I think they no longer do that. @jkwatson ?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 The proto files can be consumed as GIT submodules or copied and built directly in the consumer project.
```
Comment (bogdandrutu) [inline README.md:5]: ni: or download the zip archive, as we do in collector and java.

File: README.md (hunk 1 above)
Comment (jack-berg) [reply README.md:7]: They are still published to maven via [opentelemetry-proto-java](https://github.com/open-telemetry/opentelemetry-proto-java).

**Review by bogdandrutu — APPROVED** [review]

**Comment by blumamir** [comment]
Should JS also be added to the README under [Generate gRPC Client Libraries](https://github.com/open-telemetry/opentelemetry-proto#generate-grpc-client-libraries)?
The makefile generates those files just like any other language mentioned there.

**Comment by tigrannajaryan** [comment]
> Should JS also be added to the README under [Generate gRPC Client Libraries](https://github.com/open-telemetry/opentelemetry-proto#generate-grpc-client-libraries)? The makefile generates those files just like any other language mentioned there.

@blumamir please feel free to submit a PR that adds this.

**Pull request merged**